tools/defaults.bzl

```diff
@@ -162,6 +162,7 @@ def pkg_npm(name, pkg_deps = [], use_prodmode_output = False, **kwargs):
         exclude_prefixes = [
             "packages",  # Exclude compiled outputs of dependent packages
         ],
+        allow_overwrites = True,
```

Comment:
This needed to be added due to an update to `aspect_bazel_lib` that someone else must have done. I don't fully understand why the build didn't fail until this PR though?

Reply:
I've reverted this for now to make this PR the absolute minimum number of changes.